recipe/build.sh

```diff
@@ -1,43 +1,16 @@
 #!/bin/bash
 
-# install using pip from the whl files on PyPI
+mkdir bin
+ln -sf ${BUILD_PREFIX}/bin/{swig,ragel,yasm} bin/
 
-if [ `uname` == Darwin ]; then
-    if [ "$PY_VER" == "3.7" ]; then
-        WHL_FILE=https://pypi.org/packages/cp37/c/catboost/catboost-${PKG_VERSION}-cp37-cp37m-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.8" ]; then
-        WHL_FILE=https://pypi.org/packages/cp38/c/catboost/catboost-${PKG_VERSION}-cp38-cp38-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.9" ]; then
-        WHL_FILE=https://pypi.org/packages/cp39/c/catboost/catboost-${PKG_VERSION}-cp39-cp39-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.10" ]; then
-        WHL_FILE=https://pypi.org/packages/cp310/c/catboost/catboost-${PKG_VERSION}-cp310-cp310-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.11" ]; then
-        WHL_FILE=https://pypi.org/packages/cp311/c/catboost/catboost-${PKG_VERSION}-cp311-cp311-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.12" ]; then
-        WHL_FILE=https://pypi.org/packages/cp312/c/catboost/catboost-${PKG_VERSION}-cp312-cp312-macosx_11_0_universal2.whl
-    fi
-fi
+cmake ${CMAKE_ARGS} \
+  -DCMAKE_POSITION_INDEPENDENT_CODE=On \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCATBOOST_COMPONENTS="PYTHON-PACKAGE" \
+  .
 
-if [ `uname` == Linux ]; then
-    if [ "$target_platform" == "linux-aarch64" ]; then
-        TARGET_ARCH=aarch64
-    elif [ "$target_platform" == "linux-64" ]; then
-        TARGET_ARCH=x86_64
-    fi
+make -j${CPU_COUNT} _catboost _hnsw
 
-    if [ "$PY_VER" == "3.7" ]; then
-        WHL_FILE=https://pypi.org/packages/cp37/c/catboost/catboost-${PKG_VERSION}-cp37-cp37m-manylinux2014_${TARGET_ARCH}.whl
-    elif [ "$PY_VER" == "3.8" ]; then
-        WHL_FILE=https://pypi.org/packages/cp38/c/catboost/catboost-${PKG_VERSION}-cp38-cp38-manylinux2014_${TARGET_ARCH}.whl
-    elif [ "$PY_VER" == "3.9" ]; then
-        WHL_FILE=https://pypi.org/packages/cp39/c/catboost/catboost-${PKG_VERSION}-cp39-cp39-manylinux2014_${TARGET_ARCH}.whl
-    elif [ "$PY_VER" == "3.10" ]; then
-        WHL_FILE=https://pypi.org/packages/cp310/c/catboost/catboost-${PKG_VERSION}-cp310-cp310-manylinux2014_${TARGET_ARCH}.whl
-    elif [ "$PY_VER" == "3.11" ]; then
-        WHL_FILE=https://pypi.org/packages/cp311/c/catboost/catboost-${PKG_VERSION}-cp311-cp311-manylinux2014_${TARGET_ARCH}.whl
-    elif [ "$PY_VER" == "3.12" ]; then
-        WHL_FILE=https://pypi.org/packages/cp312/c/catboost/catboost-${PKG_VERSION}-cp312-cp312-manylinux2014_${TARGET_ARCH}.whl
-    fi
-fi
-
-$PYTHON -m pip install --no-deps $WHL_FILE
+cd catboost/python-package/
+$PYTHON mk_wheel.py --build-with-cuda=no
```

Comment:
Conda packages provided now do support CUDA and we intend to keep it this way.
If you are just experimenting with the build starting with the simpler variant without CUDA - that's ok, but if you want to merge this PR CUDA has to be supported.